lib/services/build.ts

```diff
@@ -228,6 +228,12 @@ export class BuildService implements Project.IBuildService {
 				return buildResult;
 			} else if(settings.platform === "WP8") {
 				var buildCompanyHubApp = !settings.downloadFiles;
+				if(this.$project.projectData.WPSdk === "8.1" && ((options.release && settings.downloadFiles) || settings.buildForTAM)) {
```

Comment:
will you upload debug configuration in the store?
Also our documentation says that if you want to publish your app in Windows Phone Store, you should execute $ appbuilder build wp8 --download --release
http://docs.telerik.com/platform/appbuilder/publishing-your-app/distribute-production/publish-wp8#build-and-publish-your-windows-phone-app-to-the-windows-phone-store-from-the-appbuilder-command-line-interface